contracts/utils/cryptography/ECDSA.sol

```diff
@@ -22,6 +22,10 @@ library ECDSA {
      * recover to arbitrary addresses for non-hashed data. A safe way to ensure
      * this is by receiving a hash of the original message (which may otherwise
      * be too long), and then calling {toEthSignedMessageHash} on it.
+     *
+     * Documentation for signature generation:
```

Comment:
Can you take a look at https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2190 as well? I think this is a good opportunity to review that issue, see if it's still valid with the latest version of Ganache, and document the workarounds. We're using the `fixSignature` helper in our tests (we actually may no longer need this under Hardhat...), but we should also think about what our users will need to do in their own tests and how we could make it easier for them.

Reply:
You'll like 04d4b09 :)